### PR TITLE
refactor(raw_vehicle_cmd_converter): rework parameters

### PR DESCRIPTION
**Pull request by PhoebeWu21**
## Description
Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the `raw_vehicle_cmd_converter` package.

- Remove the default value from the source code in order to ensure all parameter values are passed from the parameter files.
- Create the schema

## Tests performed
Package build and launch locally.
`colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to raw_vehicle_cmd_converter`

## Effects on system behavior
More reliable and faster parameter configuration file creation.

## Pre-review checklist for the PR author
The PR author **must** check the checkboxes below when creating the PR.
- [x] I've confirmed the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
- [x] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).

## In-review checklist for the PR reviewers
The PR reviewers **must** check the checkboxes below before approval.
- [ ] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author
The PR author **must** check the checkboxes below before merging.
- [x] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: vehicle/raw_vehicle_cmd_converter/README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 ## Parameters
 
+{{ json_to_markdown("vehicle/raw_vehicle_cmd_converter/schema/raw_vehicle_cmd_converter.schema.json") }}
+
 | Parameter                  | Type   | Description                                                                     |
 | -------------------------- | ------ | ------------------------------------------------------------------------------- |
 | `update_rate`              | double | timer's update rate                                                             |
```
[the remark this answers is not in the document]
Comment (PhoebeWu21) [reply vehicle/raw_vehicle_cmd_converter/README.md]: Got it. Done.

File: vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml [hunk 2]
```diff
@@ -1,34 +1,20 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="converter_param_path" default="$(find-pkg-share raw_vehicle_cmd_converter)/config/converter.param.yaml"/>
   <arg name="csv_path_accel_map" default="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/accel_map.csv"/>
   <arg name="csv_path_brake_map" default="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/brake_map.csv"/>
   <arg name="csv_path_steer_map" default="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/steer_map.csv"/>
```
[the remark this answers is not in the document]
Comment (PhoebeWu21) [reply vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml]: Removed.

[the remark this answers is not in the document]
Comment (ambroise-arm) [reply vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml]: Doing `ros2 launch raw_vehicle_cmd_converter raw_vehicle_converter.launch.xml` leads to:
```
[raw_vehicle_cmd_converter_node-1] Cannot open $(find-pkg-share raw_vehicle_cmd_converter)/data/default/accel_map.csv
[raw_vehicle_cmd_converter_node-1] terminate called after throwing an instance of 'std::invalid_argument'
[raw_vehicle_cmd_converter_node-1]   what():  Accel map is invalid.
```

We just need to set `allow_substs` for the param file, like in: https://github.com/autowarefoundation/autoware.universe/blob/c171d1c80c13d64da73e5cdf9c81c19df5d29e95/common/tvm_utility/launch/yolo_v2_tiny_example.launch.xml#L21

[the remark this answers is not in the document]
Comment (PhoebeWu21) [reply vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml]: Got it. Done.

[the remark this answers is not in the document]
Comment (PhoebeWu21) [reply vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml]: Got it. Done.

**Comment by PhoebeWu21** [comment]
@ambroise-arm @satoshi-ota Hi, the json schema check fails seems because to the error schema setting in behavior_velocity_planner node
image

**Comment by satoshi-ota** [comment]
@PhoebeWu21 Thanks :+1: 
I created fix PR. Could you review it?
https://github.com/autowarefoundation/autoware.universe/pull/5676

**Comment by satoshi-ota** [comment]
@PhoebeWu21 #5676 was merged.

**Review by taikitanaka3 — APPROVED** [review]
looks good

**Review by TakaHoribe — APPROVED** [review]
LGTM. Thank you for your PR.

**Pull request merged**